cms/lib/xblock/tagging/tagging.py

```diff
@@ -93,9 +93,9 @@ def save_tags(self, request=None, suffix=None):  # pylint: disable=unused-argume
 
     def get_event_context(self, event_type, event):  # pylint: disable=unused-argument
         """
-        This method return data that should be associated with the "check_problem" event
+        This method return data that should be associated with the "submit_problem" event
```

Comment:
Does this mean that the analytics event is changing? Or is the "event" just an internal detail?

We don't want to change analytics events (at least not without checking with the Analytics team and notifying researchers).